flows/contact.go

```diff
@@ -125,18 +125,18 @@ func NewContactReference(uuid ContactUUID, name string) *ContactReference {
 //------------------------------------------------------------------------------------------
 
 type contactEnvelope struct {
-	UUID        ContactUUID    `json:"uuid"              validate:"required,uuid4"`
+	UUID        ContactUUID    `json:"uuid" validate:"required,uuid4"`
 	Name        string         `json:"name"`
 	Language    utils.Language `json:"language"`
 	Timezone    string         `json:"timezone"`
 	URNs        URNList        `json:"urns"`
 	Groups      GroupList      `json:"groups"`
 	Fields      *Fields        `json:"fields,omitempty"`
-	ChannelUUID ChannelUUID    `json:"channel_uuid"      validate:"omitempty,uuid4"`
+	ChannelUUID ChannelUUID    `json:"channel_uuid,omitempty" validate:"omitempty,uuid4"`
```

Comment:
Per discussion this morning, if messages require channels, seems contacts probably should too.

Reply:
What about flows that don't send messages and are just messing with contact state? Wouldn't those contacts not have a channel?

Reply:
Ah yes, indeed. Nevermind.

Reply:
You could have a new contact that's never been contacted before, no?